specification/automation/resource-manager/Microsoft.Automation/common/v5/definitions.json

```diff
@@ -0,0 +1,109 @@
+{
+  "swagger": "2.0",
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "version": "2023-11-01",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "PrivateEndpointConnection": {
```

Comment:
>PrivateEndpointConnection [](http://example.com/codeflow?start=5&length=25)

the object seem similar to the one defined in `common-types` here - [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/privatelinks.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/privatelinks.json)

Is there a specific reason to not use it?

Reply:
Removed these Private Link related definitions from the common definitions.json as private links APIs are not part of this stable version.